scripts/wait-for-it.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
```

Comment:
Consider: https://github.com/ufoscout/docker-compose-wait

Reply:
You can do the same on travis, no?

Reply:
Perhaps I could, although I do not see an easy way. 
In any case, neither script is really what I need; what I really need in the travis container is something that checks when alchemy has finished building (i.e. not just responding on localhost:3000 but actually returning the bundle...)

also, it is not clear to me why you think having a short shell script in the repo is better than having an external dependency on a 600Kb external blob of compiled code :-)

Reply:
Well, because its not part of the project. Just like you don't download npm packages and commit them to the repo you should't commit files like this.